checkout-ui-custom/src/partials/Deliver/AddAddressManualForm.js

```diff
@@ -0,0 +1,258 @@
+      required: false,
+      type: 'hidden',
+      value: '',
+    },
+    {
+      name: 'number',
+      label: 'Street Number',
+      required: true,
+      value: '',
+      type: 'text',
+    },
+    {
+      name: 'street',
+      label: 'Street address',
+      required: true,
+      value: '',
+    },
+    {
+      name: 'addressType',
+      label: 'Address type',
+      required: true,
+      type: 'radio',
+      options: [
+        { value: 'residential', label: 'Residential', checked: true },
+        { value: 'business', label: 'Business' },
+      ],
+    },
+    {
+      name: 'businessName',
+      label: 'Business name',
+      required: false,
+      value: '',
+      maxLength: 100,
+    },
+    {
+      name: 'companyBuilding',
+      label: 'Building/Complex and number',
+      required: false,
+      value: '',
+      maxLength: 100,
+    },
+    {
+      name: 'neighborhood',
+      label: 'Suburb',
+      value: '',
+      maxLength: 750,
+      required: true,
+    },
+    {
+      name: 'city',
+      label: 'City',
+      required: true,
+      value: '',
+      maxLength: 750,
+    },
+    {
+      name: 'postalCode',
+      label: 'Postal code',
+      value: '',
+      type: 'tel',
+      minlength: 4,
+      maxLength: 4,
+      required: true,
+    },
+    {
+      type: 'note',
+      required: false,
+      name: 'suburb-postal-reminder',
+      value: 'Make sure to specify the correct Suburb and Postal code so we can easily find your address.',
+    },
+    {
+      name: 'state',
+      label: 'Province',
+      type: 'dropdown',
+      required: true,
+      options: [
+        { value: '', label: 'Select' },
+        { value: 'EC', label: 'Eastern Cape' },
+        { value: 'FS', label: 'Free State' },
+        { value: 'GP', label: 'Gauteng' },
+        { value: 'KZN', label: 'KwaZulu-Natal' },
+        { value: 'LP', label: 'Limpopo' },
+        { value: 'MP', label: 'Mpumalanga' },
+        { value: 'NC', label: 'Northern Cape' },
+        { value: 'NW', label: 'North West' },
+        { value: 'WC', label: 'Western Cape' },
+      ],
+    },
+    {
+      type: 'note',
+      required: false,
+      name: 'country-display',
+      label: 'Country',
+      value: 'South Africa',
+    },
+    {
+      type: 'hidden',
+      required: true,
+      name: 'country',
+      value: 'ZAF',
+    },
+    {
+      name: 'receiverName',
+      label: `Recipient's name`,
+      required: true,
+      value: getBestRecipient({ type: 'delivery' }),
+    },
+    {
+      name: 'complement',
+      required: false,
+      type: 'hidden',
+      value: '',
+    },
+    {
+      name: 'receiverPhone',
+      label: `Recipient's mobile number`,
+      required: true,
+      type: 'tel',
+      helperText: 'We send shipping updates to this number.',
+      minlength: 9,
+      error: 'Please enter a valid phone number',
+      containerClasses: 'custom-field-complement',
+      idOverride: ADD_FORM_RECEIVER_PHONE_ID,
```

Comment:
rename to something better

Reply:
ADD_ADDRESS_FORM_MANUAL_RECIEVER_PHONE_ID